src/chainparams.cpp

```diff
@@ -265,20 +265,27 @@ class SigNetParams : public CChainParams {
         uint32_t genesis_nonce = 0;
 
         if (!args.IsArgSet("-signet_blockscript")) {
-            throw std::runtime_error(strprintf("%s: -signet_blockscript is mandatory for signet networks", __func__));
-        }
-        if (args.GetArgs("-signet_blockscript").size() != 1) {
-            throw std::runtime_error(strprintf("%s: -signet_blockscript cannot be multiple values.", __func__));
-        }
-        bin = ParseHex(args.GetArgs("-signet_blockscript")[0]);
-        if (args.IsArgSet("-signet_genesisnonce")) {
+            LogPrintf("Using default signet network\n");
+            bin = ParseHex("512103ad5e0edad18cb1f0fc0d28a3d4f1f3e445640337489abb10404f2d1e086be43051ae");
+            genesis_nonce = 621297;
+            vSeeds.push_back("178.128.221.177");
+            vSeeds.push_back("2a01:7c8:d005:390::5");
+            vSeeds.push_back("ntv3mtqw5wt63red.onion:38333");
+        } else {
+            if (!args.IsArgSet("-signet_blockscript")) {
```

Comment:
This will never trigger as it has already been checked and handled. It can be removed.